Makefile

```diff
@@ -23,6 +23,7 @@ LINT_SOURCES = \
         test/cpp/asyncprogressworker.cpp \
         test/cpp/asyncworkererror.cpp \
         test/cpp/bufferworkerpersistent.cpp \
+	test/cpp/error.cpp \
```

Comment:
Could be. I normalized everything to tabs in 80cf724cf898ac5568b52d44b06c0d5c0ebf0764, but this commit is older. I'll rebase with master or something.